utility.py

```
@@ -1,1182 +1,83 @@
-    """
-    # Look up the emoji name from the map, falling back to using the key directly
-    actual_emoji_name = EMOJI_MAP.get(emoji_name, emoji_name)
+# ruff: noqa: F401, F403
 
-    if emoji := discord.utils.get(bot.emojis, name=actual_emoji_name):
-        return str(emoji)
-    if fallback is not None:
-        return fallback
-    return StatusIcon.INFO.value
+import logging
+import re as _re
```

Flake8 (7.3.0)

[error] 're as _re' imported but unused

(F401)